libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/TracerContextStorage.java

```diff
@@ -8,13 +8,16 @@
 
 package org.opensearch.telemetry.tracing;
 
+import org.opensearch.common.annotation.InternalApi;
+
 /**
  * Storage interface used for storing tracing context
  * @param <K> key type
  * @param <V> value type
  *
  * @opensearch.internal
  */
+@InternalApi
 public interface TracerContextStorage<K, V> {
```

Comment:
Shouldn't it be @ExperimentalApi?

Reply:
We don't expose it anywhere, and only use internally in `DefaultTracer`

Reply:
ThreadContextBasedTracerContextStorage is internal one. There could be multiple implementations provided. Now we can keep it internal also.